src/pages/Favorites/Favorites.tsx

```diff
@@ -34,6 +34,7 @@ function Favorites({ match, classes }) {
             resourceID={resource}
             expand={true}
             includeCheck={false}
+            includePhoto={false}
```

Comment:
bit confused by includePhoto, I didn't seem to see a difference between the resourcecards on Favorites.tsx and Guides.tsx !